README.md

```diff
@@ -10,6 +10,53 @@ official packaging.
 
 ## Usage
 
+### Documentation
```

Comment:
Yeah I get where you are coming from. I had to sleep on it before I came up with "Invocation", and that's not necessarily much better.